config/routes.rb

```diff
@@ -636,6 +636,7 @@
 
     get '/tasks' => 'tasks#index', as: :tasks
     post '/tasks/:task' => 'tasks#run', as: :run_task
+    get '/tasks/create-fake-provider' => 'tasks#create_fake_provider'
```

Comment:
This should be a POST as it creates data

Reply:
Ah! I started with a POST and changed it to GET because it's not sending any params 🙈. Good to know, thanks!